inc/class-registration.php

```diff
@@ -164,7 +164,7 @@ public function enqueue_assets() {
 		$asset_file = include OTTER_BLOCKS_PATH . '/build/blocks/leaflet-map.asset.php';
 		wp_register_script( 'leaflet', OTTER_BLOCKS_URL . 'assets/leaflet/leaflet.js', [], $asset_file['version'], true );
 		wp_script_add_data( 'leaflet', 'async', true );
-		wp_register_script( 'leaflet-gesture-handling', OTTER_BLOCKS_URL . 'assets/leaflet/leaflet-gesture-handling.min.js', array( 'leaflet' ), $asset_file['version'], true );
+		wp_register_script( 'leaflet-gesture-handling', OTTER_BLOCKS_URL . 'build/assets/leaflet-gesture-handling.min.js', array( 'leaflet' ), $asset_file['version'], true );
```

Comment:
@HardeepAsrani, any idea about that detection if someone loads the same script again with another plugin? This script will load only if someone uses the Maps blocks.

Reply:
@Soare-Robert-Daniel You don't need a detection, just try to load the original script again on the page as see if that's causing any conflicts. If it isn't then it's fine.

Reply:
Anyway, I'll merge it for now. We can leave it for when it happens as that's not the biggest issue.